### PR TITLE
Update ReadMe with Insert and Select Queries

### PR DESCRIPTION
**Pull request by maail**
Added in Insert and Select Queries for Supabase Swift

## What kind of change does this PR introduce?

Added documentation for Insert and Select Queries

## What is the current behavior?

Current documentation does not show basic CRUD functionalities. It's only available as tests on Postgrest-Swift repo, which not everyone might find easily.

## What is the new behavior?

Added documentation for Insert and Select Queries

## Additional context

I'll add in Update and Delete functionality once I've used them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,6 +173,69 @@ Task {
...
+    do {
+        if let response = try? await query.execute(),
+           let data = try? response.underlyingResponse.data  {
+           let returnArray = try JSONDecoder().decode([InsertModel].self, from:  data)
+            print("### Returned: \(returnArray.first)")
+        }
+    } catch {
+        print("### Insert Error: \(error)")
+    }
+}
```
Comment (grsouza) [inline README.md:212]: You don't need to manually decode the response, every `Respnse` returned has ` value` property that returns the decoded model, you just need to help Swift figure it out the type.

Also when creating the query, you can specify that you want a `single` value to be returned.

```suggestion
let query = client.database
            .from("{ Your Table Name }")
            .insert(values: insertData,
                    returning: .representation) // you will need to add this to return the added data
            .select(columns: "id") // specifiy which column names to be returned. Leave it empty for all columns
            .single() // specify you want to return a single value.
            
Task {
    do {
        let response: InsertModel = try await query.execute().value
        print("### Returned: \(response)")
    } catch {
        print("### Insert Error: \(error)")
    }
}
```

File: README.md [hunk 2]
```diff
@@ -173,6 +173,69 @@ Task {
...
+    do {
+        if let response = try? await query.execute(),
+           let data = try? response.underlyingResponse.data  {
+            let returnArray = try JSONDecoder().decode([InsertModel].self, from:  data)
+            print("### Returned: \(returnArray.first)")
+        }
+    } catch {
+        print("### Insert Error: \(error)")
+    }
+}
```
Comment (grsouza) [inline README.md:236]: Same thing as the insert example, you can define as `single` and use `value`'s response to fetch a decoded object.

**Review by grsouza — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (maail) [reply README.md:212]: Oh that's pretty cool.

File: README.md (hunk 2 above)
Comment (maail) [reply README.md]: @grsouza Updated.

**Review by grsouza — APPROVED** [review]

**Pull request merged**